coalib/misc/Shell.py

```diff
@@ -51,15 +53,18 @@ def run_interactive_shell_command(command, **kwargs):
                     command.
     """
     if not kwargs.get('shell', False) and isinstance(command, str):
-        command = shlex.split(command)
+        command, *cargs = shlex.split(command)
```

Comment:
@justuswilhelm Changes type of `command` from sequence to `str`, but only if `shell is True`, so messing up the Windows logic below...